src/Support/SupportData.php

```diff
@@ -358,20 +327,25 @@ public function get_error_log() {
 			];
 		}
 
-		$file        = file( $error_log_path );
-		$max_lines   = max( 0, count( $file ) - 200 );
-		$file_length = count( $file );
-		$contents    = [];
+		$no_of_lines = 200;
+
+		$file  = fopen( $error_log_path, 'r' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen
```

Comment:
We should suppress any errors that may be thrown upon attempting to read the file.

```suggestion
		$file  = @fopen( $error_log_path, 'r' ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_read_fopen
```

Reply:
Addressed in [39e51e6](https://github.com/ampproject/amp-wp/pull/6627/commits/39e51e68a80dc30f28b041c21f51ab49886ec180)